{
  repo.begin(1l);
  repo.acquire(1l);
  try {
    repo.acquire(1l);
    fail("Should not have been allowed access.");
  }
 catch (  ConcurrentAccessException e) {
    assertThat(e.getMessage(),equalTo("Cannot access '1', because another client is currently using it."));
  }
  repo.release(1l);
  assertThat(repo.acquire(1l),equalTo(0l));
}
